packages/components/src/components/DistributionChart.tsx

```diff
@@ -75,10 +87,32 @@ export const DistributionChart: React.FC<DistributionChartProps> = (props) => {
       );
       widthProp = 20;
     }
-    const domain = shapes.value.flatMap((shape) =>
+    const predomain = shapes.value.flatMap((shape) =>
       shape.discrete.concat(shape.continuous)
     );
 
+    const domain =
+      xAxis === "dateTime"
+        ? predomain.map((p) => {
+            return { dateTime: p.x, y: p.y };
```

Comment:
small thing, but I imagine that maybe instead of a ``return``, you could do something like

``predomain.map((p) => ({dateTime: p.x, y: p.y})``

(it's been a while since I did js like this though, might be wrong)